sequitur.go

```diff
@@ -259,7 +260,7 @@ func (pr *prettyPrinter) printTerminal(w io.Writer, sym uint64) error {
 	case '\\', '(', ')', '_', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9':
 		out = "\\" + string(sym)
 	default:
```

Comment:
I wonder if it should be a bit more careful about which characters it can print unadorned (think zero-width, combining chars, etc), but I guess since there's no hex escape option (presumably) perhaps it can't.

Reply:
I agree, but I don't think any action is required here.

Reply:
The pretty printing code was ripped from the C code for compatibility during testing.  We can obviously change it to whatever we'd like, but I'll have to up date my (not pushed) golden test output files.  I'll see if I can shrink down the test corpus into something smaller and get some better tests added to the repo.

Reply:
I'll leave it as-is in terms of functionality (but using slices rather than strings). I can see the need to escape digits so as not to confuse them in the output with nodes in the tree.

Reply:
Obviously, more tests is always better :)